packages/playground/src/utils/manual.ts

```diff
@@ -1,7 +1,7 @@
 const BASE = window.env.MANUAL_URL;
 
 export const manual = {
-  dedicated_machines: `${BASE}/documentation/dashboard/deploy/dedicated_machines.html`,
+  dedicated_machines: `${BASE}/documentation/dashboard/deploy/node_finder.html#dedicated-nodes`,
```

Comment:
please fix the replace dedicated_machines in the `billing_pricing` as well
should follow the new node_finde url  `https://www.manual.grid.tf/documentation/dashboard/deploy/node_finder.html#billing--pricing`

Reply:
done